tests/testthat/test-queuefallout.R

```diff
@@ -11,12 +11,11 @@ test_that("Validate QueueFallout using legacy credentials", {
                      metric="pageviews",
                      element="page",
                      c("http://randyzwitch.com/r-amazon-ec2", "http://randyzwitch.com/julia-import-data")
```

Comment:
You are missing the closing `)` on line 13,  and missing commas between the arguments on lines 25 and 38. Please check the validity of your code using RStudio, which will highlight any invalid syntax.